netmaster/objApi/apiController.go

```diff
@@ -880,6 +882,6 @@ func (ac *APIController) BgpDelete(bgpNeighborCfg *contivModel.Bgp) error {
 }
 
 //BgpUpdate updates bgp config
-func (ac *APIController) BgpUpdate(oldbgpNeighborCfg *contivModel.Bgp, NewbgpNeighborCfg *contivModel.Bgp) error {
+func (ac *APIController) BgpUpdate(oldbgpCfg *contivModel.Bgp, NewbgpCfg *contivModel.Bgp) error {
 	return nil
```

Comment:
why is 'update' returning 'nil'? Do we need to retrigger the updates and retrigger grat-arp, etc. to update this?